docs/schema.md

```diff
@@ -69,6 +69,12 @@ Developers can make a schema as abstract schema specifying type=abstract. The de
           title: Tenant
           type: string
           unique: false
+      indexes:
```

Comment:
Have you tried it openapi generator? If not properly handled https://github.com/cloudwan/gohan/blob/master/cli/template.go it may output invalid schema.